### DISCUSSION TITLE
BuyPass doesn't work

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1236] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
the window closes like after 2 seconds xD

**Comment by S-PScripts** [comment]
https://the-buypass.netlify.app/ (S-PScripts)
https://playful-meringue-fb58bc.netlify.app/ (S-PScripts)
https://buypass-poc.netlify.app/ (Brandon421-ops)

**Reply by Luigi-Master** [reply]
Cool, it dosen't work for sercurly but that's a nice bypass

**Reply by schoolexploitkid** [reply]
ty, now it opens

**Comment by 3kh0** [comment]
![image](https://github.com/user-attachments/assets/1c6490da-6c63-4cc1-b033-da7826fef6ad)

Idk it has something to do with this
